region/norfair/crocomire/Crocomire's Room.json

```diff
@@ -412,6 +412,26 @@
       ],
       "flashSuitChecked": true
     },
+    {
+      "link": [1, 4],
+      "name": "Crocmire Defeated, R-Mode Spark Interrupt",
+      "entranceCondition": {
+        "comeInWithRMode": {}
+      },
+      "requires": [
+        "f_DefeatedCrocomire",
+        "h_CrystalFlashForReserveEnergy",
+        "h_shinechargeMaxRunway",
+        {"autoReserveTrigger": {}},
+        "canRModeSparkInterrupt",
+        {"acidFrames": 60}
```

Comment:
with 0 tanks this took me 30 frames (44E), but I think you take acid during the reserve trigger? If so, we need a way to represent that